cmd/in-place-restart-agent/service-account-example.yaml

```diff
@@ -0,0 +1,92 @@
+# Service account
```

Comment:
I am still in favour to delegate creation of these RBAC resources to JobSet controller as discussed with @astefanutti 

That will improve UX of using that feature.